WebRoot/js/zimbraMail/prefs/view/ZmPreferencesPage.js

```diff
@@ -674,6 +674,9 @@ function(parentIdOrElem, text, width, listener, id) {
 	button.setText(text);
 	button.addSelectionListener(listener);
 	this._replaceControlElement(parentIdOrElem, button);
+	if(className) {
```

Comment:
If its about the initial comment that Dwt.delClass doesn't do anything if no params are passed, my opinion is that it doesn't harm to have a check as className could resolve to null/undefined as well. 

Suggestion: 
There could be another way to write the same code:
className && button.addClassName(className);